packaging/linux/build_binaries.sh

```diff
@@ -5,6 +5,7 @@ set -e -u -o pipefail
 here="$(dirname "$BASH_SOURCE")"
 this_repo="$(git -C "$here" rev-parse --show-toplevel)"
 kbfs_repo="$(dirname "$this_repo")/kbfs"
+updater_repo="$(dirname "$this_repo")/go-updater"
```

Comment:
We'll also need to include getting this repo and exposing it to the docker container in `docker_build.sh`.